src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -101,6 +106,11 @@ class SparqlQleverVisitor : public SparqlAutomaticVisitor {
   ExpressionPtr processIriFunctionCall(const std::string& iri,
                                        std::vector<ExpressionPtr> argList);
 
+  // TODO: Remove addVisibleVariable(const string&) when all Types use the
+  //  strong type Variable.
```

Comment:
```suggestion
  //  strong type `Variable`.
```
Convention for source code elements (types, variable names, etc.)